lib/util.go

```diff
@@ -34,3 +34,13 @@ func makeEndpointsList(headers []string) []string {
 
 	return endpoints
 }
+
+func quote(l []string) []string {
+	var quoted []string
+
+	for _, s := range l {
+		quoted = append(quoted, fmt.Sprintf("\"%s\"", s))
```

Comment:
Right, I was playing with several options and I don't know why I uploaded this one :D